docs/design/StratisSoftwareDesign.lyx

```diff
@@ -4944,1458 +4603,276 @@ See libdm/libdm-common.c _is_whitelisted_char() in the lvm2 code for more.
+\begin_inset Quotes eld
 \end_inset
-</cell>
-<cell alignment="center" valignment="top" topline="true" leftline="true" rightline="true" usebox="none">
-\begin_inset Text
-
-\begin_layout Plain Layout
-None
-\end_layout
 
+cap
+\begin_inset Quotes erd
 \end_inset
-</cell>
-</row>
-<row>
-<cell alignment="center" valignment="top" topline="true" leftline="true" usebox="none">
-\begin_inset Text
 
-\begin_layout Plain Layout
-Low
+ layer will ensure the Tier presents a single blockdev with consistent block
+ mapping for use above the tier by creating a Linear device that never relocates
+ previously-mapped block ranges.
 \end_layout
 
-\end_inset
-</cell>
-<cell alignment="center" valignment="top" topline="true" leftline="true" usebox="none">
-\begin_inset Text
-
-\begin_layout Plain Layout
-Space available is below Low threshold but above Critical
+\begin_layout Subsubsection
+Demand-based allocations
 \end_layout
 
+\begin_layout Standard
+Layers should not consume the entire space available to them when constructing
+ devices, but instead grow existing mapped allocations (or create new ones)
+ as the total demands of upper layers grow larger.
+ This is preferred over a 
+\begin_inset Quotes eld
 \end_inset
-</cell>
-<cell alignment="center" valignment="top" topline="true" leftline="true" rightline="true" usebox="none">
-\begin_inset Text
-
-\begin_layout Plain Layout
-Alert the user, run maintenance operations
-\end_layout
 
+greedy
+\begin_inset Quotes erd
 \end_inset
-</cell>
-</row>
-<row>
-<cell alignment="center" valignment="top" topline="true" bottomline="true" leftline="true" usebox="none">
-\begin_inset Text
 
-\begin_layout Plain Layout
-Critical
+ strategy because it provides a better user experience to allow the amount
+ of space allocated to both data and metadata to be calculated dynamically
+ if requirements change.
 \end_layout
 
-\end_inset
-</cell>
-<cell alignment="center" valignment="top" topline="true" bottomline="true" leftline="true" usebox="none">
-\begin_inset Text
-
-\begin_layout Plain Layout
-Space available below Critical threshold
+\begin_layout Subsection
+Operation States
 \end_layout
 
-\end_inset
-</cell>
-<cell alignment="center" valignment="top" topline="true" bottomline="true" leftline="true" rightline="true" usebox="none">
-\begin_inset Text
-
-\begin_layout Plain Layout
-Alert the user of critical state, run maintenance operations
+\begin_layout Standard
+When encountering errors, Stratis must handle them if possible, but there
+ are also errors that are severe enough to hamper Stratis's ability to function.
+ When these occur, instead of terminating, Stratis continues by transitioning
+ to a less-capable operation state.
+ This allows some measure of continued monitoring and enables its condition
+ to be visible to the user through the API.
 \end_layout
 
-\end_inset
-</cell>
-</row>
-</lyxtabular>
+\begin_layout Standard
+Stratis has operation states for the pools, meta and data extend, and also
```

Comment:
Can you get rid of this sentence? It doesn't add much and has a broken reference.